build.env

```diff
@@ -39,6 +39,7 @@ HELM_VERSION=v3.9.2
 
 # minikube settings
 MINIKUBE_VERSION=v1.26.1
+MINIKUBE_ISO_URL=https://storage.googleapis.com/minikube-builds/iso/14783/minikube-v1.26.1-1661377864-14783-amd64.iso
```

Comment:
A `FIXME: ` comment above this URL would be good, refer to the PR that contains the fscrypt changes.

Reply:
:+1: